internal/server/helper.go

```diff
@@ -40,6 +40,17 @@ func anyClientsAlive(clientInfos map[shared.ClientID]gamestate.ClientInfo) bool
 	}
 	return false
 }
+// getNonDeadClients returns a map of all clients with a non-dead status
+func getNonDeadClients(clientInfos map[shared.ClientID]gamestate.ClientInfo, 
```

Comment:
onoooo soz pls merge main i already implemented it jn

Reply:
oh no! sorry